msteams-platform/graph-api/App-permissions/Teams-app-permissions.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Manage Teams app permissions
+author: surbhigupta
+description: In this module, learn how Teams apps are managed in different places based on the feature.
+ms.topic: reference
+ms.localizationpriority: medium
+ms.author: lomeybur
+---
+
+# Teams app permissions
+
+Microsoft Teams apps are managed in different places depending on app features.
+
+:::image type="content" source="../../assets/images/teams-app-permissions.png" alt-text="The screenshot describes the different Teams app permissions.":::
+
+## Resource-specific consent
+
+Resource-specific consent (RSC) is a Microsoft Teams and Microsoft Graph API integration that enables your app to use API endpoints to manage specific resources, either Teams or chats, within an organization.
+
+RSC permissions are defined in the app manifest and not in Azure Active Directory (Azure AD). You grant consent to RSC permissions when you add the app to a team. For more information, see [enable resource-specific consent in Teams](../rsc/resource-specific-consent.md).
```

Comment:
Lets add it like this : Resource-specific permissions are only available to Teams apps installed on the Teams client and are currently not part of the Azure Active Directory (AAD) portal.

Reply:
Updated the artcile as suggested.